aip/general/0132.md

```diff
@@ -156,6 +156,13 @@ request message **should** contain a `string order_by` field.
   equivalent.
 - Subfields are specified with a `.` character, such as `foo.bar` or
   `address.street`.
+- The resulting list order **should** be based on the field type's natural
+  comparator e.g. numerics ordered numerically, strings ordered
+  lexicographically, etc. However, APIs **may** choose to use a different
+  ordering; if so, it **must** be documented in the `order_by` definition.
+  - Furthermore, [well-known][] types, like `Timestamp` and `Duration` are
+    compared as their representative type; `Timestamp` is compared as time e.g.
+    before or after, `Duration` is compared numerically e.g. larger or smaller.
```

Comment:
```suggestion
    before or after, `Duration` is compared as a quantity e.g. more or less.
```
After rereading, I think `quantity` is the better term, as a "numerical value multiple of a unit of measurement" per the ol'google. As such I think the terms `more` and `less` align a bit better. (I know, nit picks)

Reply:
Hmm, I don't have a strong opinion on this but am happy to commit it if you think its better.